apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestionTable.tsx

```diff
@@ -0,0 +1,781 @@
+  const [sorting, setSorting] = useQueryState<SortingState>(
+    'sort',
+    parseAsSortingState.withDefault(DEFAULT_SORT),
+  );
+  const [columnPinning, setColumnPinning] = useQueryState(
+    'frozen',
+    parseAsColumnPinningState.withDefault(DEFAULT_PINNING),
+  );
+
+  const [gradingStatusFilter, setGradingStatusFilter] = useQueryState(
+    'status',
+    parseAsArrayOf(parseAsStringLiteral(GRADING_STATUS_VALUES)).withDefault(
+      DEFAULT_GRADING_STATUS_FILTER,
+    ),
+  );
+  const [assignedGraderFilter, setAssignedGraderFilter] = useQueryState(
+    'assigned_grader',
+    parseAsArrayOf(parseAsString).withDefault(DEFAULT_ASSIGNED_GRADER_FILTER),
+  );
+  const [gradedByFilter, setGradedByFilter] = useQueryState(
+    'graded_by',
+    parseAsArrayOf(parseAsString).withDefault(DEFAULT_GRADED_BY_FILTER),
+  );
+  const [submissionGroupFilter, setSubmissionGroupFilter] = useQueryState(
+    'submission_group',
+    parseAsArrayOf(parseAsString).withDefault(DEFAULT_SUBMISSION_GROUP_FILTER),
+  );
+  const [aiAgreementFilter, setAiAgreementFilter] = useQueryState(
+    'ai_agreement',
+    parseAsArrayOf(parseAsString).withDefault(DEFAULT_AI_AGREEMENT_FILTER),
+  );
+  const [rubricItemsFilter, setRubricItemsFilter] = useQueryState(
+    'rubric_items',
+    parseAsArrayOf(parseAsString).withDefault([]),
+  );
+
+  const [manualPointsFilter, setManualPointsFilter] = useQueryState(
+    'manual_points',
+    parseAsNumericFilter.withDefault(''),
+  );
+  const [autoPointsFilter, setAutoPointsFilter] = useQueryState(
+    'auto_points',
+    parseAsNumericFilter.withDefault(''),
+  );
+  const [totalPointsFilter, setTotalPointsFilter] = useQueryState(
+    'total_points',
+    parseAsNumericFilter.withDefault(''),
+  );
+  const [scoreFilter, setScoreFilter] = useQueryState(
+    'score',
+    parseAsNumericFilter.withDefault(''),
+  );
+
+  const { createCheckboxProps } = useShiftClickCheckbox<InstanceQuestionRow>();
+
+  const [columnSizing, setColumnSizing] = useState<ColumnSizingState>({});
+  const [rowSelection, setRowSelection] = useState({});
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+  const [successMessage, setSuccessMessage] = useState<string | null>(null);
+  const [showDeleteAiGradingModal, setShowDeleteAiGradingModal] = useState(false);
+  const [showDeleteAiGroupingsModal, setShowDeleteAiGroupingsModal] = useState(false);
+
+  const queryClientInstance = useQueryClient();
+
+  // Fetch instance questions data
+  const { data: instanceQuestions = initialInstanceQuestions } = useQuery<InstanceQuestionRow[]>({
+    queryKey: ['instance-questions', urlPrefix, assessmentId, assessmentQuestion.id],
+    queryFn: async () => {
+      const res = await fetch(
+        `${urlPrefix}/assessment/${assessmentId}/manual_grading/assessment_question/${assessmentQuestion.id}/instances.json`,
+      );
+      if (!res.ok) throw new Error('Failed to fetch instance questions');
```

Comment:
Something is incorrect about the error handling here. I added an artificial error to `instances.json` and used the points popover to edit points. After saving, the popover remained open, there was no indication that an error occurred, and I saw in the devtools that something (presumably `useQuery` here?) seemed to retry a bunch with exponential backoff.